.github/workflows/release.yml

```diff
@@ -42,18 +42,12 @@ jobs:
         with:
           version: $RELEASE
       
-      - name: Build new images
-        run: docker compose -f docker-compose.yml -f eclipse-pass.local.yml build idp ldap
-
       - name: Acceptance tests
         if: ${{ inputs.runtests }}
         uses: ./.github/actions/acceptance-test
         with:
           pullimages: 'missing'
 
-      - name: Push release images
-        run: docker compose -f docker-compose.yml -f eclipse-pass.local.yml push idp ldap
-
       - name: Bump image versions to next development version
         uses: ./.github/actions/update-pass-version
         with:
```

Comment:
Check last line in this file, I think it can be removed.

Reply:
Will fix.

Reply:
Done

Reply:
Actually, I meant the build --push idp ldap line can be removed I think: https://github.com/eclipse-pass/pass-docker/pull/379/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R64

Reply:
Ahah!

Reply:
Done.